openc3/Gemfile

```diff
@@ -11,3 +11,5 @@ gem 'rails', '~> 7.0.0'
 gem 'bootsnap', '>= 1.9.3', require: false
 gem 'rack-cors', '~> 2.0'
 gem 'tzinfo-data'
+# httpclient hasn't been released since 2016 so use Shopify's updated one
+gem 'httpclient', git: 'https://github.com/Shopify/httpclient'
```

Comment:
Can't do this as it always requires an internet connection.

Reply:
We probably need to switch to Faraday.

Reply:
Could also use HTTP: https://github.com/httprb/http